homework-submission/myapp/src/App.js

```diff
@@ -0,0 +1,41 @@
+import React, { Component } from 'react';
+import { Route, Link, Switch } from 'react-router-dom';
+
+import StaticList from './components/Week1/index';
+import DynamicList from './components/Week2/index';
+import APICall from './components/Week3/App';
+import './App.css';
+
+class App extends Component {
+  render() {
+    return (
+      <div className='App'>
+        <ul>
+          <li>
+            <Link to='/'>Home</Link>
+          </li>
+          <li>
+            <Link to='/week1'>Week1</Link>
+          </li>
+          <li>
+            <Link to='/week2'>Week2</Link>
+          </li>
+          <li>
+            <Link to='/week3'>Week3</Link>
+          </li>
+        </ul>
+
+        <div>
+          <Switch>
+            <Route exact path='/' render={() => <h3> Homeworks </h3>} />
+            <Route path='/week1' component={StaticList} />
+            <Route exact path='/week2' component={DynamicList} />
```

Comment:
No need for an exact match here, it's only if you have a route that you could have another with the same name and different case, or you may have a root rout and you want to distinguish it from it's sub routes.

Reply:
Thank you @Meno-101 👍